.github/workflows/integration.yml

```diff
@@ -31,10 +31,33 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - {name: "PostgreSQL", task: "pg"}
-          - {name: "Tigris", task: "tigris"}
-          - {name: "Tigris main", task: "tigris", tigris_dockerfile: "tigris_main"}
-          - {name: "MongoDB", task: "mongodb"}
+          - name: "MongoDB"
+            task: "mongodb"
+            buildtags: "ferretdb_tigris"
+
+          # with pushdown
+          - name: "PostgreSQL"
+            task: "pg"
+            buildtags: "ferretdb_tigris"
+          - name: "Tigris"
+            task: "tigris"
+            buildtags: "ferretdb_tigris"
+          - name: "Tigris main"
+            task: "tigris"
+            buildtags: "ferretdb_tigris"
+            tigris_dockerfile: "tigris_main"
+
+          # without pushdown
+          - name: "PostgreSQL (no pushdown)"
+            task: "pg"
+            buildtags: "ferretdb_tigris,ferretdb_nopushdown"
+          - name: "Tigris (no pushdown)"
+            task: "tigris"
+            buildtags: "ferretdb_tigris,ferretdb_nopushdown"
+          - name: "Tigris main (no pushdown)"
+            task: "tigris"
+            buildtags: "ferretdb_tigris,ferretdb_nopushdown"
+            tigris_dockerfile: "tigris_main"
```

Comment:
I don't understand the purpose of not using pushdown

Reply:
The reason was to do extra cross-testing, but that's already covered by our testing against MongoDB